docs/intro.rst

```diff
@@ -0,0 +1,60 @@
+
+.. py:currentmodule:: rticonnextdds_connector
+
+Introduction to RTI Connector
+=============================
+
+RTI Connext DDS
+~~~~~~~~~~~~~~~
+
+Connext DDS is a software connectivity framework for real-time distributed
+applications. It uses the publish-subscribe communications model to make
+data distribution efficient and robust. At its core Connext DDS is the world's
+leading ultra-high performance, distributed networking databus.
+
+RTI Connext DDS provides programming APIs in C, C++, Java, and .NET.
```

Comment:
This could be seen as confusing. If this is the first thing you read about connector-py and you see that RTI Connext DDS APIs are in C, etc.
Maybe add Pro somewhere?